pallets/loans/src/benchmarking.rs

```diff
@@ -36,28 +37,33 @@ const PENDING_MARKET_MOCK: Market = {
 
 const INITIAL_AMOUNT: u32 = 500_000_000;
 
-fn transfer_initial_balance<T: Config>(caller: T::AccountId)
-where
+fn transfer_initial_balance<
+    T: Config + pallet_assets::Config<AssetId = AssetIdentifier, Balance = Balance>,
```

Comment:
Great work! What I don't understand is that Loans doesn't implement pallet_assets:Config if there is no inheritance. How does this work？

Reply:
this is for limiting runtime used for benchmarks, `mock` and all runtimes implemented `pallet_assets::Config` so it worked